src/components/contact-admin/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { AppContext } from '@edx/frontend-platform/react';
+import { MailtoLink } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const ContactAdmin = ({
+  children = 'Contact your administrator',
```

Comment:
Curious about this default function argument, if it's already set as a default prop below? Related, the one place this component is used right now passes a node to `children`, rendering the default unnecessary. Is the idea that this component will be re-used elsewhere? If so, should we remove the `children` from its current usage to just get the default configured?

Reply:
Yeah, there are a few places where I would like to reuse this component, removing the unnecessary bits.